### ISSUE TITLE
Additional Samples which break PortEx

### ISSUE THREAD

**Issue by drorwolmer** [katjahahn/PortEx#7]
Hello,

Here are more samples which break Portex.
They all break with the following error

```
Error: Negative seek offset
java.io.IOException: Negative seek offset
	at java.io.RandomAccessFile.seek(Native Method)
	at com.github.katjahahn.parser.IOUtil.loadBytesSafely(IOUtil.java:91)
	at com.github.katjahahn.tools.ReportCreator.readBytes$1(ReportCreator.scala:262)
	at com.github.katjahahn.tools.ReportCreator.com$github$katjahahn$tools$ReportCreator$$getResourceString$1(ReportCreator.scala:264)
	at com.github.katjahahn.tools.ReportCreator$$anonfun$manifestReport$1.apply(ReportCreator.scala:270)
	at com.github.katjahahn.tools.ReportCreator$$anonfun$manifestReport$1.apply(ReportCreator.scala:268)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at com.github.katjahahn.tools.ReportCreator.manifestReport(ReportCreator.scala:268)
	at com.github.katjahahn.tools.ReportCreator.resourcesReport(ReportCreator.scala:252)
	at com.github.katjahahn.tools.ReportCreator.specialSectionReports(ReportCreator.scala:79)
	at com.github.katjahahn.tools.ReportCreator.printReport(ReportCreator.scala:92)
	at com.github.katjahahn.tools.PortExAnalyzer$.invokeCLI(PortExAnalyzer.scala:94)
	at com.github.katjahahn.tools.PortExAnalyzer$.main(PortExAnalyzer.scala:67)
	at com.github.katjahahn.tools.PortExAnalyzer.main(PortExAnalyzer.scala)
```

SHA256 of the samples
```
ef1a1284ee19db6e047e0c1647c96c216aefb31fda312e54de28c57e63b04e00
a88979a5b289023451bb4a6b43027cc8c23e480095eb454fbf7cd945ef4f6e36
07c303c82ad8fd496a4bc2f4759086882acb8c842a6f97758d748ec0a0338ae7
05e81d6d9317e56a70631a00892f85e707c735e8207c24b6c91daa4a4b8e0c42
cca7fe00f4117ca45115e411f9a129a24396a8f231b7163933dc35a8cf49a089
04a90c8d61d249d71b1d9ad27f813e085dcda352e3990022622c6c2ec1825888
```

Here are download links for the following files:
* https://goo.gl/WEWY0C
* http://goo.gl/2qBlHW

I'm really glad to help creating the best PE Analyzer out there!

**Comment by katjahahn** [comment]
Thank you for your submission. The manifest resource entry has a bad value and causes this problem. I will add error checking to avoid this issue.

**Comment by katjahahn** [comment]
Solved with latest commit 59d2b6a9ac67bd76f138d6b79fe29fe58305becb
PortexAnalyzer.jar is also updated. Thank you for your report!
